content/blog/2022-05-03-e2e-cv-pt1.md

```diff
@@ -0,0 +1,307 @@
+---
+title:
+  'End-to-End Computer Vision API, Part 1: Data Versioning and ML pipelines'
+date: 2022-05-03
+description: >
+
+  In the last few years, training a well-performing Computer Vision (CV) model
+  in Jupyter Notebooks became fairly straightforward if you use pre-trained Deep
+  Learning models and high-level libraries that abstract away much of the
+  complexity (`fastai`, `keras`, `pytorch-lightning` are just a few
+  examples).  The hard part is still incorporating this model into an
```

Comment:
Extremely long sentence. Not easy to read/comprehend. Important as it's the abstract/summary. Try to break it up 🙂

Reply:
In general `description` should probably be much shorter since it get's rendered in the blog's front page and truncated heavily, especially with such a long title (see https://dvcorg-e2ecvblogpart1.gtsb.io/blog -- both featured post and cards below).

Reply:
done